frontend/package-lock.json

```diff
@@ -852,19 +838,15 @@
           }
         },
         "@babel/highlight": {
```

Comment:
@michaelwoodruffdev / @ccarterlandis : I think I may have gotten this wrong. It looks like there's some things in slack-config that are deleted that I should have copied into the merge.

Reply:
I will defer to @michaelwoodruffdev / @gabe-heim